src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3944,7 +4021,11 @@ namespace FanCoilUnits {
 				Node( InletNode ).MassFlowRateMinAvail = AirMassFlowLow;
 				FanFlowRatio = LowSpeedRatio;
 				Calc4PipeFanCoil( FanCoilNum, ZoneNum, FirstHVACIteration, QUnitOutMaxLS );
-				FanElecPowerLS = FanElecPower;
+				if ( FanCoil( FanCoilNum ).FanType_Num != DataHVACGlobals::FanType_SystemModelObject ) {
+					FanElecPowerLS = Fans::GetFanPower( FanCoil( FanCoilNum ).FanIndex );
+				} else {
+					FanElecPowerLS = HVACFan::fanObjs[ FanCoil( FanCoilNum ).FanIndex ]->fanPower();
```

Comment:
Just returning a value.  I removed the ugly global FanElecPower from everywhere and it gets replaced by this.